go.mod

```diff
@@ -31,6 +31,6 @@ require (
 	k8s.io/client-go v0.19.2
 	k8s.io/code-generator v0.19.2
 	sigs.k8s.io/controller-runtime v0.6.3
```

Comment:
there's also 0.6.4 and 0.7.0 for this. I think at least the 0.6.4 bump should be easy